src/components/app/App.tsx

```diff
@@ -42,14 +42,16 @@ const theme = createMuiTheme({
 
 // These wrapper components are temporary components to pass their children the value of contexts using the useContext hook
 // TODO: refactor PatchContainer and BuildView to be functional components that consume their respective contexts
-const Patches = () => {
+const Patches = (props: any) => {
+  const { params } = props.match;
```

Comment:
I'm confused as to what this is used for. Would it be possible to give props a concrete type?